kctf-operator/pkg/utils/utils.go

```diff
@@ -1,12 +1,3 @@
 // Functions that are useful
 
 package utils
```

Comment:
Do we still need this file? Or do you want to keep it for future functions?

Reply:
I created it to put functions that are common to different controllers (from the moment that we have multiple controllers). But currently we don't use it, so I can delete it.